### PR TITLE
added graph filters

### PR DESCRIPTION
**Pull request by kalyanijagdale**
# Peer Review Information

Added filters using label names for object count and visible pixels.
# Pull Request Check List

- [ ] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

**Comment by Saurav-D** [comment]
Resolve merge conflict. Merge from master.

**Comment by BlairLee** [comment]
You need to assign reviewers for this PR.

File: datasetinsights/visualization/app.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    css_file = os.path.join(this_dir, "stylesheet.css")
+    app = dash.Dash(
+        __name__,
+        external_stylesheets=[css_file],
+        suppress_callback_exceptions=True,
+    )
+    return app
+
+
+_app = _init_app()
```
Comment (Saurav-D) [inline datasetinsights/visualization/app.py]: can this be initialized inside `get_app` method?

File: datasetinsights/visualization/app.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def get_app():
```
Comment (Saurav-D) [inline datasetinsights/visualization/app.py]: Do we need this? Can we directly call _init_app?

File: datasetinsights/dashboard.py [hunk 3]
```diff
@@ -64,14 +67,26 @@ def main_layout(data_root):
...
+    if value == "dataset_overview":
+        return overview.html_overview(data_root)
+    elif value == "object_detection":
         return render_object_detection_layout(data_root)
 
 
+@app.callback(
+    Output("data_root_value", "children"), [Input("dropdown", "value")]
+)
+def clean_data(value):
```
Comment (adason) [inline datasetinsights/dashboard.py]: should be renamed to `get_data_root`?

File: datasetinsights/stats/visualization/object_detection.py [hunk 4]
```diff
@@ -203,7 +213,7 @@ def _generate_figures_lighting(self):
             self.lighting,
             x="x_rotation",
             y="y_rotation",
-            title="Light orientations",
+            # title="Light orientations",
```
Comment (adason) [inline datasetinsights/stats/visualization/object_detection.py]: Why is title commented out here? Should it be removed?

File: datasetinsights/stats/visualization/object_detection.py [hunk 5]
```diff
@@ -390,7 +404,7 @@ def _generate_figures_orientation(self, orientation):
             x="x_rot",
             y="y_rot",
             z="z_rot",
-            title="Object orientations",
+            # title="Object orientations",
```
Comment (adason) [inline datasetinsights/stats/visualization/object_detection.py]: Why is title commented out here? Should it be removed?

File: datasetinsights/dashboard.py [hunk 6]
```diff
@@ -64,14 +67,26 @@ def main_layout(data_root):
...
 )
-def render_content(tab):
-    if tab == "dataset_overview":
-        return overview(data_root)
-    elif tab == "object_detection":
+def render_content(value):
+    """ Method for rendering dashboard layout based
+        on the selected tab value."""
+    if value == "dataset_overview":
+        return overview.html_overview(data_root)
```
Comment (adason) [inline datasetinsights/dashboard.py:99]: It's still confusing how the local variable `data_root` is available here. Can you make sure to read this value by calling `get_data_root` method? Can you add comments in code to explain this a little bit?

**Review by adason — CHANGES_REQUESTED** [review]

**Review by adason — APPROVED** [review]

**Pull request merged**